### ISSUE TITLE
Getting started: "Home" link at the bottom of page is not descriptive

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#64]
URL: https://hcdev.openplus.ca/en/documentation/getting-started
User Agent: Chrome
Language: Both

Issue Description: For each page of the "Help Manual", there is a "Home" link at the bottom of the page. The link text "Home" is not descriptive. The tooltip for the link is "Go to front page" for both English and French pages which needs to be more specific. English tooltip on the French page.
![image](https://user-images.githubusercontent.com/88776404/129077413-2e007c90-a508-4378-96ea-b521a54e7939.png)
![image](https://user-images.githubusercontent.com/88776404/129077445-248d9a6d-de9c-4a94-827d-fcadd8d88d46.png)

Steps to reproduce: Navigate to the "Home" link, and select it.

Expected results: A more descriptive link text should be used for the "Home" link so that the purpose of the link can be determined from the link text alone.

**Comment by echouin** [comment]
@XiuJuanXia this is completed. It is currently on Dev and will be on live version of website (12 Mar 22):
![Home (3) - Copy](https://user-images.githubusercontent.com/88099454/157706058-43138f8e-d603-4a25-954a-e06e51a98f0b.png)
![home2 (2) - Copy](https://user-images.githubusercontent.com/88099454/157706061-2566a57d-141c-4d42-815f-6e1ad96d8931.png)

**Comment by XiuJuanXia** [comment]
Confirmed this issue is fixed.
